src/mount/mount.ceph.c

```diff
@@ -425,24 +425,41 @@ static void ceph_mount_info_free(struct ceph_mount_info *cmi)
 	free(cmi->cmi_conf);
 }
 
-static int finalize_options(struct ceph_mount_info *cmi)
+static int append_secret_option(struct ceph_mount_info *cmi)
 {
-	int pos;
+	int pos = strlen(cmi->cmi_opts);
+	const char *key_name = cmi->cmi_name;
+	const char *secret = cmi->cmi_secret;
 
```

Comment:
These locals seem redundant.  cmi->cmi_secret and cmi->cmi_name aren't that much longer to type.